### DISCUSSION TITLE
Japanese inline input not working in WezTerm with fcitx5 and i3wm

### DISCUSSION THREAD

**Discussion by j-o-lantern0422** [wezterm/wezterm#3553] [Q&A] [answered]
Hello,

I am using WezTerm with fcitx5 (Mozc) for Japanese input on my Arch Linux system running i3wm. I am experiencing an issue where inline Japanese input is not working in WezTerm. I can input Japanese characters, but they do not appear inline as I type.

Here is a summary of what I have tried so far:

1. Installed the necessary packages for fcitx5, including fcitx5, fcitx5-configtool, fcitx5-mozc, fcitx5-gtk, and noto-fonts-cjk.
2. Configured the environment variables correctly, including GTK_IM_MODULE, QT_IM_MODULE, and XMODIFIERS.
3. Confirmed that fcitx5 and Mozc are working correctly in other applications (e.g., xterm) and that WezTerm is up-to-date.
4. Tried changing the rendering backend from OpenGL to WebGPU, but it resulted in Japanese input not working at all.

Is there a known compatibility issue between WezTerm and fcitx5 or any additional settings I should try to enable inline Japanese input?

Any help would be greatly appreciated. Thank you!

**Comment by wez** [comment, marked as answer]
Possibly the same issue as https://github.com/wez/wezterm/issues/3411

**Comment by j-o-lantern0422** [comment]
Probably the same problem. I put in the fcitx5 settings that the questioner mentioned in the discussion over there and it seemed to solve the same problem. Thank you!
